docs/api.rst

```diff
@@ -1600,61 +1600,61 @@ of :class:`enum.Enum`.
 
     .. attribute:: text
 
-        A text channel.
+        A text channel. Value = 0.
```

Comment:
I'm really not sure if these are necessary, they don't seem to be included in any other enums.

Reply:
Yeah, looking back on this it was not necessary at all. It was mostly a misunderstanding of what issue #1510 wanted, and the real "improvement" here is just in the part where the Enum docstring is read by Option, and the change in Option behavior so that it will accept an inputted "description" text, instead of trying to overwrite it with whatever it finds in the underlying Class (including Enums, which were empty, but will not provide a description from the docstring).
I can post sample bot code and take some screenshots to demonstrate the improvement later, or remove the "Value = 0" text from the Enums documentation after work, if you want.